### PR TITLE
Activating supply beacons makes the supply console do a noise

### PR DESCRIPTION
**Pull request by lbnesquik**
## About The Pull Request
When you activate a beacon, it will do a noise to notify any nearby RO that something changed regarding the beacon list. 
I would have put an inverse noise, but I didn't find any that was good enough
## Why It's Good For The Game
Makes it easier for req workers to know when that dang beacon is down, PFC jim!
## Changelog
:cl:
qol: Activating supply beacons notify the shipside console with a noise
/:cl:

### CODE REVIEW

File: code/game/objects/machinery/squad_supply/supply_console.dm [hunk 1]
```diff
@@ -34,9 +35,15 @@
 			supply_pad = _supply_pad
 			return
 
+
+/obj/machinery/computer/supplydrop_console/proc/ping_beacon()
```
Comment (Lumipharon) [inline code/game/objects/machinery/squad_supply/supply_console.dm:39]: autodoc

File: code/game/objects/machinery/squad_supply/supply_console.dm [hunk 2]
```diff
@@ -34,9 +35,15 @@
...
+	SIGNAL_HANDLER
+	playsound(src,'sound/machines/terminal_prompt_confirm.ogg', 50, TRUE)
+
 /obj/machinery/computer/supplydrop_console/Destroy()
 	supply_beacon = null
 	supply_pad = null
+	UnregisterSignal(SSdcs, COMSIG_GLOB_SUPPLY_BEACON_CREATED)
```
Comment (Lumipharon) [inline code/game/objects/machinery/squad_supply/supply_console.dm:46]: sigs are already unregistered on destroy, you don't need to specify it.

**Review by Lumipharon — CHANGES_REQUESTED** [review]

File: code/game/objects/machinery/squad_supply/supply_console.dm (hunk 2 above)
Comment (lbnesquik) [reply code/game/objects/machinery/squad_supply/supply_console.dm:46]: Oh, neat. The AI has a lot of those global unregisters. Should I nuke them?

File: code/game/objects/machinery/squad_supply/supply_console.dm (hunk 1 above)
Comment (lbnesquik) [reply code/game/objects/machinery/squad_supply/supply_console.dm:39]: Done

**Comment by QualityVan** [comment]
This seems like it should have a cooldown.

**Comment by lbnesquik** [comment]
> This seems like it should have a cooldown.

The supply beacon activation has a cooldown built in.

File: code/game/objects/machinery/squad_supply/supply_console.dm (hunk 2 above)
Comment (Lumipharon) [reply code/game/objects/machinery/squad_supply/supply_console.dm:46]: yes, but not in this pr

Comment (lbnesquik) [reply code/game/objects/machinery/squad_supply/supply_console.dm:46]: Done

File: code/__DEFINES/dcs/signals.dm [hunk 3]
```diff
@@ -67,6 +67,9 @@
 #define COMSIG_GLOB_CAS_LASER_CREATED "!cas_laser_sent"
 #define COMSIG_GLOB_RAILGUN_LASER_CREATED "!railgun_laser_sent"
 
+//Signals for fire support
```
Comment (Lumipharon) [inline code/__DEFINES/dcs/signals.dm:70]: bruh what is this copy past comment? Also make them (both) autodoc, not a comment

File: code/game/objects/machinery/squad_supply/supply_console.dm (hunk 2 above)
Comment (Lumipharon) [reply code/game/objects/machinery/squad_supply/supply_console.dm:46]: uh, you still need to remove this.

**Review by Lumipharon — CHANGES_REQUESTED** [review]

**Pull request merged**